src/app/children/view-schools-component/edit-school-dialog/edit-school-dialog.component.html

```diff
@@ -10,7 +10,7 @@ <h2 mat-dialog-title>{{creating ? 'Add' : 'Edit'}} school for {{child.name}}</h2
 </mat-form-field>
 
 <mat-form-field>
-  <input matInput [(ngModel)]='childSchoolRelation.schoolClass' placeholder='Class' title='class' type='text'>
+  <input matInput [(ngModel)]='childSchoolRelation.class' placeholder='Class' title='class' type='text'>
```

Comment:
`class` is or may be a protected keyword of TypeScript, that's why I renamed the attribute to `schoolClass` initially.

Reply:
Oh sorry I dind't know that. I was not sure, what the current version of it was.

Reply:
Funny enough, I never saw any issues with using `class` but it's probably better to avoid it.